lore/settings.py

```diff
@@ -289,6 +291,13 @@ def get_var(name, default):
     },
 }
 
+REST_FRAMEWORK = {
```

Comment:
We can change this in a separate PR worst case, right? The page size came from @Ferdi. We don't really use `DEFAULT_PERMISSION_CLASSES` since we override it always but I figured it was a good precaution